indico/htdocs/sass/partials/_actionboxes.scss

```diff
@@ -84,11 +84,19 @@
     background: $background-color;
     color: $color;
 
-    > .section > .icon {
+    > .section {
```

Comment:
I agree with removing it from the `.section` class, but not from the `.icon` below. Otherwise it would affect any icon element in the `.action-box`.